frameworks/kafka/docs/limitations.md

```diff
@@ -13,7 +13,9 @@ The "disk" configuration value is denominated in MB. We recommend you set the co
 
 ## Security
 
-The security features introduced in Apache Kafka 0.9 are not supported at this time.
+### Kerberos
+
+When Kerberos is enabled, the broker vip is disabled as it will not function correctly.
```

Comment:
We may want to be slightly careful with the language here so it's clear which component doesn't like working with VIPs.  Maybe a brief discussion of Kerberos' behavior would be good.